### DISCUSSION TITLE
Help Getting The Weather Channel Stream

### DISCUSSION THREAD

**Discussion by MapGuy11** [iptv-org/community#408] [Q&A]
After The Weather Channel shut down their streaming page. Finally sadly the stream URL has gone 403. I am trying to decipher the stream URL from their Android TV app. I am not the best but I have found this.

1. Requests Are Coming from this Akamai Server: 104.114.74.136 (HTTPS though so you can't trace stream in WireShark unless I am dumb and you can)
2. Here a working apk file to decipher: [https://m.apkpure.com/the-weather-channel-app/com.weather.alps/download?from=details](https://m.apkpure.com/the-weather-channel-app/com.weather.alps/download?from=details)
3. If anyone can get the URL that would be awesome!

**Comment by Dum4G** [comment]
>HTTPS though so you can't trace stream in WireShark unless I am dumb and you can

AFAIK you can make all your traffic to go with your own certificate to make it possible

**Reply by MapGuy11** [reply]
Yeah I tried still can't get it! Maybe someone with better skills then me can decipher it.

**Reply by BellezaEmporium** [reply]
A bit of Frida and some scripts that deactivates SSL pinning might be used to find where does the scene take place.

**Comment by Tribalize** [comment]
for testing. https://weather.newscdn.org/IK_X7_RSeodkAuGdWcxHKA==,1628025995/1253851229/tracks-v1a1/mono.m3u8

**Reply by MapGuy11** [reply]
@Tribalize But aren't these tokened and die?

**Reply by Dum4G** [reply]
Yes they are

**Reply by Tribalize** [reply]
Didn't know if the smarter people might make use of it.

**Comment by ciuly33** [comment]
În lun., 15 mai 2023 la 23:09 Aleksandr Statciuk ***@***.***>
a scris:
